wiki/Gameplay/Score/osu!catch/fr.md

```diff
@@ -1,18 +1,13 @@
----
-outdated: true
-outdated_since: c78e8f94260067c49d36a55deaaf7f40cb796b39
----
-
 # Système de notation d'osu!catch
 
-*Voir aussi : [Système de jugement d'osu!catch](/wiki/Gameplay/Judgement/osu!catch)*
+*Voir également : [Système de jugement d'osu!catch](/wiki/Gameplay/Judgement/osu!catch)*
 
 Le score pour osu!catch utilise le même multiplicateur qu'[osu!](/wiki/Game_mode/osu!).
-Cependant, le fruit s'ajoute différemment au score.
+Cependant, le fruit qui additionne le score est différent de celui d'osu! 
 
 - Un fruit normal donne un score de 300, multiplié par le multiplicateur.
 - Un juice drop (slider tick) donne un score de 100.
-- Un droplet (slider path) donne un score de 10.
+- Le plus petit drop (également appelé "droplet", ou slider trail/path) donne un score de 10.
```

Comment:
```suggestion
- Le plus petit drop (également appelé "droplet", ou slider trail/path) donne un score de 10.
```

C'est pas `slider tail` au lieu de `slider trail` ?

Reply:
![image](https://user-images.githubusercontent.com/50368878/167678666-0172cbd1-a99f-44b3-a911-f5a28f078618.png)

Reply:
@Walavouchey Is there a mistake here? I don't think `slider trail` exists

Reply:
this is osu!catch, although i guess it should be called "juice stream" instead, since that's how it's referred to in the code

![image](https://user-images.githubusercontent.com/36758269/167766843-77f6c7db-9710-4afa-8010-8158b5439fd5.png)

Reply:
and no we're not talking about "slider tails" here

Reply:
https://osu.ppy.sh/wiki/en/Hit_object/Juice_stream